infrastructure/foreman-configuration/foreman-server-instance-user-data.tpl.sh

```diff
@@ -77,8 +77,8 @@ docker run \\
        -e DATABASE_PASSWORD=${database_password} \\
        -v /tmp:/tmp \\
        -it ${dockerhub_repo}/dr_\"\$1\" python3 manage.py \"\$2\"
-" >> /home/ubuntu/run_cron_job_test.sh
-chmod +x /home/ubuntu/run_cron_job_test.sh
+" >> /home/ubuntu/run_cron_job.sh
+chmod +x /home/ubuntu/run_cron_job.sh
```

Comment:
call it run_container_manage_command or something similar, you can pass in the non-dr-prefixed container name and then the name of the mangement command ex: `run_container_manage_command [container] [command]`

If you want to change it I just think you should call it what it does, not named after who is calling it.

Reply:
Done.